src/model/wallet.ts

```diff
@@ -930,14 +1056,39 @@ const migrateSecrets = async (): Promise<MigratedSecretsResult | null> => {
       return null;
     }
 
+    let userPIN = null;
+    if (android) {
```

Comment:
Android users should never be able to reach this code given that they should never need to migrateSecrets